cpp-httplib.yaml

```diff
@@ -0,0 +1,39 @@
+# Generated from https://git.alpinelinux.org/aports/plain/main/cxxopts/APKBUILD
+package:
+  name: cpp-httplib
+  version: 0.16.3
+  epoch: 0
+  description: C++ header-only HTTP/HTTPS server and client library
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - cmake
+      - ninja
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/yhirose/cpp-httplib.git
+      tag: v${{package.version}}
+      expected-commit: af56b7ec0ba9d18b788ed064f76d936eb52c5db5
+      recurse-submodules: true
+
+  - runs: |
```

Comment:
```
- uses: cmake/configure
- uses: cmake/build
- uses: cmake/install
```

Should work

Reply:
This worked. `melange convert` suggests autoconf pipelines and I ran into issues there. I'll try to check out what the issue with that is.